### DISCUSSION TITLE
How can I deploy the backend to the vercel?

### DISCUSSION THREAD

**Discussion by melody4482** [vercel/community#2605] [Help] [answered]
### Summary

Hi everyone, thanks to vercel.
I deployed many React and Next.js Projects to that but I can't deploy MERN stack.
Is anybody knows about this one?
Please help me.

### Steps to Reproduce

React is good for deployment but I can't deploy the ExpressJS

**Comment by vighnesh153** [comment]
Why not use NextJS lambdas?

**Comment by amyegan** [comment, marked as answer]
We have a guide for using Express.js with Vercel which I think will help: https://vercel.com/guides/using-express-with-vercel
